web/src/app/selection/DestinationField.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { DestinationType, FieldValueInput } from '../../schema'
+import DestinationInputDirect from './DestinationInputDirect'
+import { useDestinationType } from '../util/RequireConfig'
+import DestinationSearchSelect from './DestinationSearchSelect'
+import { Grid } from '@mui/material'
+
+export type DestinationFieldProps = {
+  value: FieldValueInput[]
+  onChange?: (value: FieldValueInput[]) => void
+  destType: DestinationType
+
+  disabled?: boolean
+
+  destFieldErrors?: DestFieldError[]
+}
+
+export interface DestFieldError {
+  fieldID: string
+  message: string
+}
+
+export default function DestinationField(
+  props: DestinationFieldProps,
+): React.ReactNode {
+  const dest = useDestinationType(props.destType)
+
+  return (
+    <Grid container spacing={2}>
+      {dest.requiredFields.map((field) => {
+        const fieldValue =
+          (props.value || []).find((v) => v.fieldID === field.fieldID)?.value ||
+          ''
+
+        function handleChange(newValue: string): void {
+          if (!props.onChange) return
+
+          const newValues = (props.value || [])
+            .filter((v) => v.fieldID !== field.fieldID)
+            .concat({
+              fieldID: field.fieldID,
+              value: newValue,
+            })
+
+          props.onChange(newValues)
+        }
+
+        const fieldErrMsg =
+          props.destFieldErrors?.find((err) => err.fieldID === field.fieldID)
+            ?.message || ''
+
+        if (field.isSearchSelectable)
+          return (
+            <Grid key={field.fieldID} item xs={12} sm={12} md={6}>
```

Comment:
Is the intent to have these side by side or do we want them always stacked like before?

![Screenshot 2024-01-16 at 11 07 00 AM](https://github.com/target/goalert/assets/11381794/cdc4b352-9461-4ca2-9773-7a12cb38161b)

Reply:
The intention was to have them side by side.

Reply:
For now, this will only come into play with the schedule notification form IIRC.

When that work is being done, it may need to be tweaked, but for now I think this just came from the user contact method form as a starting point.

Reply:
Gotcha, I thought this was being made to simulate the multi fields in the admin config (which are stacked), so I wanted to check. If this is going to stay as is feel free to resolve.

Reply:
We updated it to be stacked. Screenshot has been updated.